pkgs/development/python-modules/pocketsphinx/default.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  cmake,
+  cython,
+  fetchFromGitHub,
+  memory-profiler,
+  ninja,
+  pathspec,
+  pocketsphinx,
+  pytestCheckHook,
+  scikit-build,
+  scikit-build-core,
+  sounddevice,
+}:
+
+buildPythonPackage rec {
+  inherit (pocketsphinx) version src;
+  pname = "pocketsphinx";
+  pyproject = true;
+
+  dontUseCmakeConfigure = true;
+
+  env.CMAKE_ARGS = lib.cmakeBool "USE_INSTALLED_POCKETSPHINX" true;
+
+  buildInputs = [ pocketsphinx ];
+
+  build-system = [
+    cmake
+    cython
+    ninja
+    pathspec
```

Comment:
`pathspec` doesn't seem to be used, but is it necessary?
https://github.com/search?q=repo%3Acmusphinx%2Fpocketsphinx+pathspec&type=code

Reply:
Build fails when `pathspec` is not included.

```log
> * Getting build dependencies for wheel...
>
> ERROR Missing dependencies:
>        scikit-build-core
>      pathspec>=0.10.1
```